apps/src/code-studio/LegacyDialog.js

```diff
@@ -66,10 +66,10 @@ var LegacyDialog = (module.exports = function (options) {
 
   var close = options.close === undefined ? true : options.close;
 
-  var closeLink = $('<div id="x-close"/>')
+  var closeLink = $('<button id="x-close"/>')
     .addClass('x-close')
     .attr('data-dismiss', 'modal');
-  this.div = $('<div tabindex="-1"/>').addClass('modal');
+  this.div = $('<div tabindex="0"/>').addClass('modal');
```

Comment:
Could `tabindex` be removed? Is the `<div>` being focused or is the `<button>` being focused?

Reply:
I double-checked and reversed the change. The edit was not necessary. 

The tabindex is also not necessary for the functionality either, so I can remove it too.